docs/guides/run-jobs-batch.mdx

```diff
@@ -50,4 +50,29 @@ The following example shows how you can partition a long list of circuits into m
 
 <Admonition type="caution">
     If you set `backend=backend` in a V2 primitive, the program is run in job mode, even if it's inside a batch or session context. Setting `backend=backend` is deprecated as of IBM Qiskit Runtime 0.24.0.  Instead, use the `mode` parameter.
+</Admonition>
+
+<span id="batch-details"></span>
+## Determine batch details
+
+For a comprehensive overview of a batch's configuration and status, use the  `batch.details() method`.
+
+``` python
+from qiskit_ibm_runtime import QiskitRuntimeService, SamplerV2 as Sampler
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+with Batch(backend=backend) as batch:
+    sampler = Sampler()
+    for partition in partitions:
+        job = sampler.run(partition)
+print(batch.details())
```

Comment:
Note strictly necessary, but it makes more sense to access the `batch` object inside the batch context. 

```suggestion
        job = sampler.run(partition)
        print(batch.details())
```